src/console.cc

```diff
@@ -0,0 +1,163 @@
+#include "console.h"
+
+int s21::Console::Run() {
+  while (true) {
+    std::cout << messages.at(ConsoleMessage::kMenu);
+    try {
+      switch (InputItem()) {
+        case MenuItem::kQuit: {
+          std::cout << "Exit.\n";
+          return 0;
+        }
```

Comment:
Можно проверить на EOF на всякий случай. Неосторожный пользователь может жмякнуть Ctrl + D.

```suggestion
      switch (InputItem()) {
        case EOF:
        case MenuItem::kQuit: {
          std::cout << "Exit.\n";
          return 0;
        }
```